lms/djangoapps/certificates/api.py

```diff
@@ -452,3 +452,41 @@ def emit_certificate_event(event_name, user, course_id, course=None, event_data=
 
     with tracker.get_tracker().context(event_name, context):
         tracker.emit(event_name, event_data)
+
+
+def get_certificate_header():
+    """
+    Return data to be used in Certificate Header,
+    data returned should be customized according to the microsite settings
+    """
+    data = dict(
+        logo_src=get_logo_url(),
+        logo_url=branding_api.get_base_url(),
+    )
+
+    return data
+
+
+def get_certificate_footer():
+    """
+    Return data to be used in Certificate Footer,
+    data returned should be customized according to the microsite settings
+    """
+    data = dict()
+
+    # get Terms of Service and Honor Code page url
+    terms_of_service_and_honor_code = branding_api.get_tos_and_honor_code_url()
+    if terms_of_service_and_honor_code != branding_api.EMPTY_URL:
```

Comment:
Hmm, what do you think about moving get_logo_url from __init__.py to api.py as part of this PR?  This would be aligned with our general technical strategy and there appear to be only two templates which reference the get_logo_url function, once each -- /lms/templates/navigation.html and lms/templates/navigation-edx.html